content/applications/finance/payment_providers/stripe.rst

```diff
@@ -9,9 +9,9 @@ Link your Stripe Account with Odoo
 ==================================
 
 .. seealso::
-   - :ref:`payment_acquirers/add_new`
+   - :ref:`payment_providers/add_new`
 
-The method to acquire your credentials depends on your hosting type:
+The method to provide your credentials depends on your hosting type:
```

Comment:
actually, I hadn't paid enough attention in other documents, but this change isn't correct...
It's probably the only one, though.
@hote-odoo @Demesmaeker

Reply:
thank you !